src/utils/logger.js

```diff
@@ -37,4 +21,31 @@ const logger = winston.createLogger({
   rejectionHandlers: transports,
 });
 
+if (process.env.SMTP_PASSWORD) {
+  if (config.get('logger.sendMailOnError')) {
+    transports.push(new winston.transports.Mail({
+      to: config.get('logger.sendMailOnError.to'),
+      from: config.get('logger.sendMailOnError.from'),
+      host: config.get('logger.smtp.host'),
+      username: config.get('logger.smtp.username'),
+      password: process.env.SMTP_PASSWORD,
+      ssl: true,
+      timeout: 30 * 1000,
+      formatter: args => args[Object.getOwnPropertySymbols(args)[1]], // Returns the full error message, the same that is visible in the console. It is referenced in the argument object with a Symbol of which we do not have the reference to but we know it is the second one.
+      exitOnError: true,
+      level: 'error',
+      subject: `[OTA] [Federated API] Error — ${os.hostname()}`,
+    }));
+
+    logger.configure({
+      transports,
+      rejectionHandlers: transports,
+    });
+  } else {
+    logger.warn('Configuration key "logger.sendMailOnError" was not found; the mailer feature of the Logger module won\'t be enabled');
+  }
+} else {
+  logger.warn('Environment variable "SMTP_PASSWORD" was not found; the mailer feature of the Logger module won\'t be enabled');
```

Comment:
I believe we should log this warning only if `logger.sendMailOnError` is set up: if there is no intention to send an email anyway, then the missing env variable is not an issue.